models/base_hook.py

```diff
@@ -0,0 +1,36 @@
+from typing import Dict, Optional
+
+from pydantic import BaseModel, Field
+
+from models.consts import WebhookEventType
+
+
+class Account(BaseModel):
+    id: int = Field(..., description="Account ID", example=1)
+    name: str = Field(..., description="Account owner name", example="John")
+    nick: str = Field(..., description="Account nick", example="Test")
```

Comment:
It's not an user's account in `meta`. So, it makes sense to add docstring to the model, and `example="John"` is wrong